docs/source/main_classes/trainer.rst

```diff
@@ -773,7 +817,7 @@ If you don't configure the ``optimizer`` entry in the configuration file, the :c
 automatically set it to ``AdamW`` and will use the supplied values or the defaults for the following command line
 arguments: ``--learning_rate``, ``--adam_beta1``, ``--adam_beta2``, ``--adam_epsilon`` and ``--weight_decay``.
 
-Here is an example of the pre-configured ``optimizer`` entry for AdamW:
+Here is an example of the pre-configured ``optimizer`` entry for ``AdamW``:
```

Comment:
```suggestion
Here is an example of the pre-configured ``optimizer`` entry for :class:`~transformers.AdamW`:
```

Reply:
This is DS AdamW :)

I know the docs are a bit hairy at the moment. Will need to take a fresh look once the ZeRO-3 dusts settles.

Reply:
Thank you for having another look, @sgugger!